package.json

```diff
@@ -54,6 +55,7 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-minify": "^0.2.0",
```

Comment:
For some reason this dev-dependency is needed when webpack bundles files-saver. It's listed as a preset in file-savers' babelrc file.  Seems odd to have to include it here but I couldn't find a way around `Error: Couldn't find preset "minify" relative to directory "/Users/georgebabey/dev/edx-portal"`